BuildForSDG/Team-226-Frontend
# Ft reset password #4

## Description
To test visit
/reset-password
/confirm-password

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #4 

## How Has This Been Tested?
yarn test


## Checklist:
